testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java

```diff
@@ -324,6 +324,13 @@ public void createDuplicatedUser7() {
         assertAdminEvents.assertEmpty();
 
     }
+    
+    // KEYCLOAK-7015
```

Comment:
Do you really need to add comments on test to show from which "Jira issue" they come from?

Reply:
Sometimes it's helpful.  It's a simple way to document the deep reasoning for writing that particular test.

Reply:
hmm. Thanks for the feedback.
Do we need a reason to write unit tests?

Reply:
yes